Maddori.Apple/Maddori.Apple/Screen/Setup/SetNickname/SetNicknameViewController.swift

```diff
@@ -109,6 +114,16 @@ final class SetNicknameViewController: BaseViewController {
     
     // MARK: - life cycle
     
+    init(from: ViewType) {
+        self.fromView = from
+        super.init()
+    }
+    
+    @available(*, unavailable)
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
```

Comment:
p2;

```suggestion
    @available(*, unavailable)
    required init?(coder: NSCoder) {
        nil
    }
```

요기도 마찬가지입니당

Reply:
[561f6f9](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Maddori.Apple/pull/328/commits/561f6f9b1a4a93b5fe611118433fa737418a4e15)

완료!

Reply:
[5f087b0](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Maddori.Apple/pull/328/commits/5f087b02c080391174134490ca128dcc3177d379)